examples/storybooks/index.js

```diff
@@ -18,11 +18,22 @@ import TreeDataIOExample from './tree-data-io';
 import TreeToTreeExample from './tree-to-tree';
 import styles from './generic.scss';
 
+import { handleClick, SANDBOX_URL } from './sandbox-utils';
+
 const wrapWithSource = (node, src) => (
   <div>
     {node}
 
     <br />
+    <form id="codesandbox-form" action={SANDBOX_URL} method="POST">
+      <input id="codesandbox-parameters" type="hidden" name="parameters" />
+    </form>
+    <button
+      className={styles.sandboxLink}
+      onClick={handleClick(src)}
+    >
+      VIEW SANDBOX →
```

Comment:
Hmm... I'm trying to think of a phrase that makes it obvious what this link does, without requiring them to know what codesandbox is. Here are some ideas:
PLAY WITH THIS CODE →
OPEN IN PLAYGROUND →